verification/curator-service/ui/src/components/landing-page/ChangePassword.test.tsx

```diff
@@ -0,0 +1,67 @@
+import { render, screen, waitFor } from '../util/test-utils';
+import LandingPage from './LandingPage';
+import userEvent from '@testing-library/user-event';
+import { act } from 'react-dom/test-utils';
+import { vi } from 'vitest';
+
+describe('<ChangePasswordForm />', () => {
+    beforeAll(() => {
+        vi.stubEnv(
+            'RECAPTCHA_SITE_KEY',
+            '6LeIxAcTAAAAAJcZVRqyHh71UMIEGNQ_MXjiZKhI',
```

Comment:
checking that it is okay to include this in a public repo?

Reply:
It's ok, site key is supposed to be public, we already have it in other places in code.